<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sign_up</class>
 <widget class="QDialog" name="Sign_up">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Signup Page</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image.qrc">
    <normaloff>:/images/Screenshot (264).png</normaloff>:/images/Screenshot (264).png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#Sign_up{
border-image:url(:/images/sign.png);
}
QLineEdit{
background-color:transparent;
border : 3px solid transparent;
	font: 14pt &quot;MS Shell Dlg 2&quot;;
border-radius:20px;
font:bold;
padding-left:10px;
color:white;
	border-bottom-color: white;
}</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>190</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>270</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>350</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>430</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>510</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>591</y>
     <width>371</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;
font:bold;
color:white;
border-radius:20px;
border:2px solid white;
background-color: rgb(188, 63, 0);</string>
   </property>
   <property name="text">
    <string>SUBMIT </string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>486</x>
     <y>710</y>
     <width>701</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	font: 75 15pt &quot;MS Shell Dlg 2&quot;;
   color:white;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
